Dockerfile

```diff
@@ -62,6 +62,20 @@ COPY config config
 COPY devtools devtools
 RUN npm install
 
+# Fetch external Bazel artifacts.
+# Copy over the WORKSPACE file, everything it imports, and bazelisk.
+COPY tools/bazelisk* tools/
+COPY build_defs/emscripten build_defs/emscripten
+COPY build_defs/kotlin_native build_defs/kotlin_native
+COPY emscripten_cache emscripten_cache
```

Comment:
I don't think we're using this anymore. Can you test that we get away with the build w/o copying this over? In another PR, we'll likely get rid of this; so not extending it to the dockerimage would help with its removal.

Reply:
Good point. We don't need any of these directories as we are only fetching maven for now.